.github/workflows/gradlew-update.yaml

```diff
@@ -0,0 +1,17 @@
+name: Gradle update
+
+on:
+  schedule:
+    - cron: '0 4 * * *'
+
+jobs:
+  update:
+    runs-on: ubuntu-20.04
```

Comment:
imo using a specific version here instead of `ubuntu-latest` will require more maintenance in the future than dealing with breakages when they occur. I think whichever direction we go on this should be uniform across the org. Currently we seem to be using `ubuntu-latest` everywhere, but we should have a discussion about changing if you feel it's worthwhile.

Reply:
I believe using a specific version is the right way but I agree that we should be consistent. Changed it for now.